core/src/main/java/com/radixdlt/bootstrap/RadixNodeBootstrapper.java

```diff
@@ -135,27 +145,45 @@ public void shutdown() {
     }
   }
 
+  private final ECDSASecp256k1PublicKey selfPublicKey;
   private final Network network;
+  private final Addressing addressing;
   private final Hasher hasher;
   private final RuntimeProperties properties;
   private final GenesisFromPropertiesLoader genesisFromPropertiesLoader;
   private final GenesisStore genesisStore;
+  private final File nodeStorageDir;
 
   @Inject
   public RadixNodeBootstrapper(
+      @Self ECDSASecp256k1PublicKey selfPublicKey,
       Network network,
+      Addressing addressing,
       Hasher hasher,
       RuntimeProperties properties,
       GenesisFromPropertiesLoader genesisFromPropertiesLoader,
-      GenesisStore genesisStore) {
+      GenesisStore genesisStore,
+      @NodeStorageLocation String nodeStorageLocation) {
+    this.selfPublicKey = selfPublicKey;
     this.network = network;
+    this.addressing = addressing;
     this.hasher = hasher;
     this.properties = properties;
     this.genesisFromPropertiesLoader = genesisFromPropertiesLoader;
     this.genesisStore = genesisStore;
+    this.nodeStorageDir = new File(nodeStorageLocation);
   }
 
   public RadixNodeBootstrapperHandle bootstrapRadixNode() {
+    log.info("Radix node {} is starting...", addressing.encodeNodeAddress(selfPublicKey));
```

Comment:
Could we maybe clarify that it's bootstrapping or booting, or that the bootstrapper is starting? To clarify the separation between this start-up and the main node startup?

Or else ensure the log on main start-up mentions that genesis has been located, and the core is now starting?

Reply:
:+1: I'll clarify that in a follow up PR